tests/components/template/test_alarm_control_panel.py

```diff
@@ -76,6 +77,12 @@ async def test_template_state_text(hass):
     state = hass.states.get("alarm_control_panel.test_template_panel")
     assert state.state == STATE_ALARM_ARMED_NIGHT
 
+    hass.states.async_set("alarm_control_panel.test", STATE_ALARM_ARMING)
+    await hass.async_block_till_done()
+
+    state = hass.states.get("alarm_control_panel.test_template_panel")
+    assert state.state == STATE_ALARM_ARMING
```

Comment:
Codacy found an issue: [Use of assert detected. The enclosed code will be removed when compiling to optimised byte code.](https://app.codacy.com/manual/foreign-sub/home-assistant/pullRequest?prid=6333224)